omnisafe/algos/__init__.py

```diff
@@ -12,9 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
+"""init"""
 
+algo_type = {
+    'off-policy': ['DDPG'],
+    'on-policy': [
+        'CPO',
+        'FOCOPS',
+        'CPPOPid',
+        'FOCOPS',
+        'NaturalPG',
+        'NPGLag',
+        'PCPO',
+        'PDO',
+        'PolicyGradient',
+        'PPO',
+        'PPOLag',
+        'TRPO',
+        'TRPOLag',
+    ],
+    'model-based': ['MBPPOLag', 'SafeLoop'],
+}
```

Comment:
Constants should use `ALL_CAPS` names. Also, this may place after the import statements.